gossip/src/epoch_slots.rs

```diff
@@ -225,50 +226,35 @@ impl CompressedSlots {
 }
 
 #[derive(Serialize, Deserialize, Clone, Default, PartialEq, Eq, AbiExample)]
-pub struct EpochSlots {
-    pub from: Pubkey,
-    pub slots: Vec<CompressedSlots>,
-    pub wallclock: u64,
+pub struct CompressedSlotsVec {
+    slots: Vec<CompressedSlots>,
```

Comment:
Can you remove this new type and just use `Vec<CompressedSlots>` in `RestartLastVotedForkSlots`?

We can refactor common code once the code is finalized and stable.
No need to do that in this PR while also modifying `EpochSlots`.

Reply:
Changed.